portfolio/src/main/webapp/index.html

```diff
@@ -8,14 +8,83 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <h2>Theodora-Mara Pislar</h2>
-      <p>This is my portfolio.</p>
-      <p>Here you will find more things about me.</p>
+      <h1 style = "text-align : center;">My Portfolio</h1>
```

Comment:
This is surely going to work (i.e. the text in this header will be centered), but have you seen the advice in https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns?